website/config/policies.js

```diff
@@ -55,4 +55,5 @@ module.exports.policies = {
   'view-upgrade': true,
   'deliver-premium-upgrade-form': true,
   'view-compliance': true,
+  'demo/view-okta-webflow': true,
```

Comment:
How's about:
-  `demo/*` 
- and add a comment explaining what "demo/" is for (internal Fleet demos)
- if time allows, rename this to `experimental/*` instead, and the other files  (I know it's annoying, but that way there's less likely to be naming collision issues in the future)

Reply:
Makes sense! I changed the page to be at `/experimental/okta-webflow` and updated the policy to be for `experimental/*`